package.json

```diff
@@ -7,11 +7,12 @@
 		"eslint-config-prettier": "^8.6.0",
 		"eslint-plugin-prettier": "^4.2.1",
 		"prettier": "^2.8.4",
-		"typescript": "^4.9.5"
+		"typescript": "^4.9.5",
+		"vite": "^4.1.4",
+		"vitest": "^0.29.2"
 	},
 	"scripts": {
 		"format": "prettier --write .eslintrc.cjs && eslint --quiet --fix --ext .cjs,.ts .eslintrc.cjs",
-		"test:run": "vitest run",
```

Comment:
This was giving this error message: 
![image](https://user-images.githubusercontent.com/342922/223492397-2457d2ec-12de-4eb6-8496-48ebb8e4c18b.png)